src/transformers/models/bit/modeling_bit.py

```diff
@@ -620,15 +708,15 @@ def forward(
         )
 
 
-# Copied from transformers.models.resnet.modeling_resnet.ResNetPreTrainedModel with ResNet->Bit,resnet->bit
+# Copied from transformers.models.resnet.modeling_resnet.ResNetPreTrainedModel with ResNet->Bit,resnet->resnetv2
 class BitPreTrainedModel(PreTrainedModel):
     """
     An abstract class to handle weights initialization and a simple interface for downloading and loading pretrained
     models.
     """
 
     config_class = BitConfig
-    base_model_prefix = "bit"
+    base_model_prefix = "resnetv2"
```

Comment:
Can be updated to bit

Reply:
Fixed in [6795f0c](https://github.com/NielsRogge/transformers/pull/48/commits/6795f0c0df52cf3644dd2b2d5e36b140be5678a1)